docs/api-table.js

```diff
@@ -11,24 +11,24 @@ export const react = {
   Button: {
     required: [],
     props: [
-      ['label', 'string', '', 'Interchangeable with the default slot for labelling'],
-      ['href', 'string', '', 'When set, an anchor tag will be used instead of a button'],
-      ['type', 'string', 'button', "Controls the button's type, unused when `href` is present"],
-    ],
-    slots: [
-      ['default', 'can be used instead of the label prop for button content']
+      ['className', 'string', '', 'Additional classes to include'],
+      ['onClick', '(e: MouseEvent<HTMLButtonElement, MouseEvent>) => void', '', 'Action to be called when the component is clicked'],
```

Comment:
Should we/should not include onClick in the props list? since it's not a design system prop but instead an event handler that does not affect the appearance of the button:)

Reply:
Removed it for now

Reply:
Yeah, I saw that's how it looked in fabric as well. But I looked at other design systems and they don't include it, e.g tailwind. But let's have a discussion later with the team and take a decision, this is good for now:)